setup.cfg

```diff
@@ -4,7 +4,6 @@ logging-clear-handlers=1
 eval-attr="!=slow"
 
 [flake8]
-min-version = 2.7
```

Comment:
Reverted it in 10ce4fa8c57f18b28eaca45055429a5636a1970b but GitHub hasn't acknowledged the change on this comment thread 😒.